t_stress/test_stress.py

```diff
@@ -701,4 +701,84 @@ def test(self):
         self.assertEqual(0, client.returncode)
 
 
+@marks.parameterize_class(
+    [
+        {
+            "name": "PingFlood",
+            "clients": [
+                {
+                    "id": "ctrl_frames_flood",
+                    "type": "external",
+                    "binary": "ctrl_frames_flood",
+                    "ssl": True,
+                    "cmd_args": "-address ${tempesta_ip}:443 -threads 4 -connections 100 -debug 1 -ctrl_frame_type ping_frame -frame_count 100000",
+                },
+            ],
+        },
+        {
+            "name": "SettingsFlood",
+            "clients": [
+                {
+                    "id": "ctrl_frames_flood",
+                    "type": "external",
+                    "binary": "ctrl_frames_flood",
+                    "ssl": True,
+                    "cmd_args": "-address ${tempesta_ip}:443 -threads 4 -connections 100 -debug 1 -ctrl_frame_type settings_frame -frame_count 100000",
+                },
+            ],
+        },
+        {
+            "name": "RstFlood",
+            "clients": [
+                {
+                    "id": "ctrl_frames_flood",
+                    "type": "external",
+                    "binary": "ctrl_frames_flood",
+                    "ssl": True,
+                    "cmd_args": "-address ${tempesta_ip}:443 -threads 4 -connections 100 -debug 1 -ctrl_frame_type rst_stream_frame -frame_count 100000",
+                },
+            ],
+        },
+    ]
+)
+class TestCtrlFrameFlood(tester.TempestaTest, check_memleak=True):
+    """
+    Test stability against comtrol frames frame flood.
+    """
+
+    backends = [
+        {
+            "id": "nginx",
+            "type": "nginx",
+            "port": "8000",
+            "status_uri": "http://${server_ip}:8000/nginx_status",
+            "config": NGINX_CONFIG,
+        }
+    ]
+
+    tempesta = {
+        "config": """
+        listen 443 proto=h2;
+
+        server ${server_ip}:8000;
+
+        tls_certificate ${tempesta_workdir}/tempesta.crt;
+        tls_certificate_key ${tempesta_workdir}/tempesta.key;
+        tls_match_any_server_name;
+        cache 0;
+    """
+    }
+
+    @dmesg.limited_rate_on_tempesta_node
+    def test(self):
+        client = self.get_client("ctrl_frames_flood")
+
+        self.start_all_servers()
+        self.start_tempesta()
+        self.start_all_clients()
+        x = self.wait_while_busy(client)
+        print(x)
+        client.stop()
+
```

Comment:
Maybe should you add an assert here? For example: check that the responses are received